pyproject.toml

```diff
@@ -11,6 +11,7 @@ scrapy = "^2.11.0"
 boto3 = "^1.28.78"
 jsonschema = "^4.19.2"
 scrapelib = "^2.2.0"
+pytz = "^2023.3.post1"
```

Comment:
I needed to install pytz to run the spider

Reply:
Yes, right. I installed it using pip not poetry so this issue wasn't deployed to the github.